### PR TITLE
Adding google ads source

### PR DESCRIPTION
**Pull request by Luishfs**
# Summary
Adding google ads source to data integration pipelines

# Tests

cc:
@wangxiaoyou1993

### CODE REVIEW

File: mage_integrations/mage_integrations/sources/google_ads/templates/config.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "oauth_client_secret": "",
+    "refresh_token": "",
+    "start_date": "2020-01-01T00:00:00Z",
+    "login_customer_ids": [
+        {
+            "customerId": "",
+            "loginCustomerId": null
+        }
+    ]
+}
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/sources/google_ads/templates/config.json:13]: nit: add empty line at the end of file

File: requirements.txt [hunk 2]
```diff
@@ -56,6 +56,7 @@ snowflake-connector-python==2.7.9
 sshtunnel==0.4.0
 tables==3.7.0
 trino==0.321.0
+-e git+https://github.com/mage-ai/google-ads-python#egg=google-ads
```
Comment (wangxiaoyou1993) [inline requirements.txt:59]: Can you add this to the requirements.txt under `mage_integrations` module?

File: mage_integrations/mage_integrations/sources/google_ads/templates/config.json (hunk 1 above)
Comment (Luishfs) [reply mage_integrations/mage_integrations/sources/google_ads/templates/config.json]: Fixed on commit bd83786638ad42f1c9a6ce0cef1916c37fe312ce

File: requirements.txt (hunk 2 above)
Comment (Luishfs) [reply requirements.txt]: Fixed on commit bd83786638ad42f1c9a6ce0cef1916c37fe312ce

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**